portal/app/settings/base.py

```diff
@@ -121,3 +121,8 @@
 }
 SUN_TECH_DRIVE_URL = ''
 SUN_TECH_DRIVE_TEST_URL = 'https://test.suntechdrive.com'
+
+EGAUGE_URL='https://egauge46613.egaug.es/cgi-bin/egauge'
+EGAUGE_USER = 'owner'
+EGAUGE_PASSWORD = 'SuXbc58vi9'
```

Comment:
Maybe take the psswd out

Reply:
took password and user out and put them on gain_debug_with_secrets.py file